superset/utils/backports.py

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Shouldn't we be using https://pypi.org/project/backports.strenum/ or something similar?

Reply:
No, I tested `backports.strenum` and there are mypy issues